base_frame: world
tool_frame: flange

link3:
  # molle di "repulsione"
  spring: [0.12,0.12,0.12]
  damper: [0.001,0.001,0.001]
  activation_distance: 0.5 #metri
  #molle di "attrazione"
  self_spring: [10.5,10.5,10.5]
  self_damper: [0.001,0.001,0.001]

link5:
  # molle di "repulsione"
  spring: [0.125,0.125,0.125]
  damper: [0.001,0.001,0.001]
  activation_distance: 0.5 #metri
  #molle di "attrazione"
  self_spring: [10.25,10.25,10.25]
  self_damper: [0.001,0.001,0.001]

flange:
  # molle di "repulsione"
  spring: [0.13,0.13,0.13]
  damper: [0.001,0.001,0.001]
  activation_distance: 0.5 #metri
  #molle di "attrazione"
  self_spring: [10,10,10]
  self_damper: [0.001,0.001,0.001]

joint_spring: [1.2,1.2,1.2,0.12,0.12,0.12]
joint_damper: [1.6,1.6,1.6,0.4,0.4,0.4]

max_joint_deflaction: [1,1,1,1,1,1] #[0.05,0.1,0.15,0.2,0.25,0.3]

max_reduction: 0.5
upper_limit_repulsion: 1.0 #0.9
lower_limit_repulsion: 0.0 #0.1
